bin/system-install

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+unset CDPATH
+. "$(cd `dirname $0`/..; pwd)/bin/logstash.lib.sh"
+setup
+
+if [ -z "$1" ]; then
+  [ -r ${LOGSTASH_HOME}/config/startup.options ] && . ${LOGSTASH_HOME}/config/startup.options
+  [ -r /etc/logstash/startup.options ] && . /etc/logstash/startup.options
```

Comment:
Yes, this is what is intended.  This is necessary because the package includes LOGSTASH_HOME as /usr/share/logstash, but we want to use what's in /etc/logstash first.